mysql-test/suite/galera/t/MDEV-30418.test

```diff
@@ -0,0 +1,24 @@
+--source include/galera_cluster.inc
+
+# Save original auto_increment_offset values.
+--let $node_1=node_1
+--let $node_2=node_2
+--source ../galera/include/auto_increment_offset_save.inc
+
+--connection node_2
+select @@wsrep_slave_threads;
+SET @cluster_address_orig = @@wsrep_cluster_address;
+SET GLOBAL wsrep_cluster_address=AUTO;
```

Comment:
I'm not very familiar with wsrep. Why does setting to AUTO put the node into a disconnected state?

Reply:
Because AUTO is not correct node address see https://mariadb.com/kb/en/galera-cluster-system-variables/#wsrep_cluster_address